record/index.html.in

```diff
@@ -19,9 +19,10 @@
                     </h1>
                     <h3 id="page-subtitle" class="pull-left">
                         <a ng-href="{{::ctrl.toRecordset()}}">
-                            <span ng-if="::reference.displayname.isHTML" ng-bind-html="ctrl.bookmarkDisplayName()" ng-attr-tooltip-placement="right" ng-attr-uib-tooltip="{{::reference.table.comment ? reference.table.comment : undefined}}"></span>
-                            <span ng-if="::!reference.displayname.isHTML" ng-bind="ctrl.bookmarkDisplayName()" ng-attr-tooltip-placement="right" ng-attr-uib-tooltip="{{::reference.table.comment ? reference.table.comment : undefined}}"></span>
+                            <span ng-if="::reference.displayname.isHTML" ng-bind-html="reference.displayname.value" ng-attr-tooltip-placement="right" ng-attr-uib-tooltip="{{::reference.table.comment ? reference.table.comment : undefined}}"></span>
+                            <span ng-if="::!reference.displayname.isHTML" ng-bind="reference.displayname.value" ng-attr-tooltip-placement="right" ng-attr-uib-tooltip="{{::reference.table.comment ? reference.table.comment : undefined}}"></span>
                         </a>
+                        <span ng-if="reference && reference.location.version" tooltip-placement="bottom-left" uib-tooltip="{{::ctrl.tooltip.versionTime}} {{ctrl.versionDate()}}">({{ctrl.versionDisplay()}})</span>
```

Comment:
Should this one and the span that we're showing in share modal be the same thing (and use the same logic)? If we do, then you can turn this into a directive and use it in both places. You just need to pass the timestamp and then it will return the span with appropriate tooltip.

Reply:
The text that's shown is different and the tooltips are different. The function is called inline here where with the share case, the function is called before the dialog is opened.